src/map/battle.c

```diff
@@ -4604,8 +4604,9 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 	bool n_ele = false; // non-elemental
 	short hitpercbonus = 0;
 
-	struct map_session_data *sd, *tsd;
 	struct Damage wd;
+	struct map_session_data *sd = BL_CAST(BL_PC, src);
+	struct map_session_data *tsd = BL_CAST(BL_PC, target);
```

Comment:
better move whole this lines below to removed lines
better define vars closer to usage location

Reply:
So should I just revert it back to the original?

Reply:
no. move your two new lines of code into location where you removed lines

Reply:
Done!